packaging/fedora/snapd.spec

```diff
@@ -182,7 +183,6 @@ BuildRequires:  libtool
 BuildRequires:  gcc
 BuildRequires:  gettext
 BuildRequires:  gnupg
-BuildRequires:  indent
```

Comment:
Since we're going to be building with EPEL8, if this doesn't exist, someone should request the EPEL 8 branch so that it exists.

Reply:
Actually, `indent` is already being introduced in EPEL8: https://bodhi.fedoraproject.org/updates/FEDORA-EPEL-2019-891ee8a7be

Reply:
It's not needed for building snapd. I don't declare that in Arch either.

Reply:
I'm pretty sure it's used in the tests for `snap-confine`?

Reply:
It's disabled because indent output was unstable across versions.

Reply:
Okay then...